upload_parrots.py

```diff
@@ -19,17 +20,35 @@ def __init__(self, slack_team):
         self.browser = webdriver.Chrome()
         self.slack_team = slack_team
 
-    def login(self, username, password):
+    def login(self, username, password, google):
         if self.loggged_in:
             return
 
         self.browser.get('https://{}.slack.com/signin'.format(self.slack_team))
-        username_field = self.browser.find_element(value="email")
-        password_field = self.browser.find_element(value='password')
 
-        username_field.send_keys(username)
-        password_field.send_keys(password)
-        username_field.submit()
+        if google == True:
+            signin_button = self.browser.find_element_by_partial_link_text("Sign in with Google")
+            signin_button.click()
+
+            username_field = WebDriverWait(self.browser, 30).until(EC.presence_of_element_located((By.ID, "identifierId")))
+            username_field.send_keys(username)
+            username_next = self.browser.find_element_by_id("identifierNext")
+            username_next.click()
+
+            password_field = WebDriverWait(self.browser, 30).until(EC.presence_of_element_located((By.NAME, "password")))
+            password_field.send_keys(password)
+            password_next = self.browser.find_element_by_id("passwordNext")
+            password_next.click()
+        elif google == False:
+            username_field = self.browser.find_element(value="email")
+            password_field = self.browser.find_element(value='password')
+
+            username_field.send_keys(username)
+            password_field.send_keys(password)
+            username_field.submit()
+        else:
```

Comment:
This condition would actually never happen because `google` is always going to be a boolean